docs/models/openai.md

```diff
@@ -366,6 +366,37 @@ agent = Agent(model)
 ...
 ```
 
+### GitHub Models
+
+To use [GitHub Models](https://docs.github.com/en/github-models), you'll need a GitHub personal access token with the `models: read` permission.
+
+Once you have the token, you can use it with the [`GitHubModelsProvider`][pydantic_ai.providers.github_models.GitHubModelsProvider]:
+
+```python
+from pydantic_ai import Agent
+from pydantic_ai.models.openai import OpenAIModel
+from pydantic_ai.providers.github_models import GitHubModelsProvider
+
+model = OpenAIModel(
+    'xai/grok-3-mini',  # GitHub Models uses prefixed model names
+    provider=GitHubModelsProvider(api_key='your-github-token'),
+)
+agent = Agent(model)
+...
+```
+
+You can also set the `GITHUB_TOKEN` environment variable:
+
+```bash
+export GITHUB_TOKEN='your-github-token'
+```
+
+GitHub Models supports various model families with different prefixes:
```

Comment:
Can we link to their model index instead of listing them here?

Reply:
Done, good idea.